File: atcoder/abc244D_swap_hats.py
# uninhm
# https://atcoder.jp/contests/abc244/tasks/abc244_d
# greedy

s = list(input().split())
t = list(input().split())

x = 0
for i in range(len(s)):
    if s[i] == t[i]:
        x += 1

if x == 1:
    print('No')
else:
    print('Yes')

File: atcoder/abc244D_swap_hats.nim
# Vicfred
# https://atcoder.jp/contests/abc244/tasks/abc244_d
# implementation
import sequtils, strutils, tables, algorithm

let s = stdin.readLine.split
let t = stdin.readLine.split

var fixed : int

for i in 0..2:
  if s[i] == t[i]:
    fixed += 1

if fixed == 1:
  echo "No"
else:
  echo "Yes"
